linux_os/guide/services/base/service_cockpit_disabled/rule.yml

```diff
@@ -14,8 +14,6 @@ rationale: |-
 
 severity: medium
 
-platform: machine
```

Comment:
Containers have services too. Reverting this change.

Reply:
Most containers (all that don't use ubi-init as baseimage) don't have systemd, but they run the service that they have been configured to do by the respective Dockerfile. Checking for this would require a new check even in the OVAL standard, i.e. no SCAP scanner can check for services enabled in containers.
The question is what would be the benefit of checking for it, as it is outside of the standard.